components/authn-service/constants/constants.go

```diff
@@ -20,4 +20,4 @@ const UnassignedProjectID = "(unassigned)"
 const AllProjectsExternalID = "*"
 
 // TestPgURL is the URL to connect to the authn test DB
-const TestPgURL = "postgresql://postgres@127.0.0.1:5432/authn_test?sslmode=disable"
+const TestPgURL = "postgresql://postgres@127.0.0.1:5432/authn_test?sslmode=disable&timezone=UTC"
```

Comment:
Should this change be applied to docs/examples, too?
Specfically:
components/session-service/examples/config.yml
components/teams-service/examples/config.yml
components/teams-service/README.md

Reply:
Thanks for the pointer, these should be nuked 😃